app/views/admin/visit_requests/_waiting_list.slim

```diff
@@ -0,0 +1,14 @@
+.six.wide.column
+  table#visit-requests.ui.compact.selectable.grey.table
+    thead
+      tr
+        th colspan="3"
+          |
+            Waiiting list
```

Comment:
1. Typo
2. This text should be in .yml
3. use t('visit_requests.waiting_list')
4. Change  visit_requests.waiting_list' in `en.yml` to `Waiting list`